core/controllers/topics_and_skills_dashboard.py

```diff
@@ -117,7 +117,11 @@ def post(self):
         """Handles POST requests."""
         name = self.payload.get('name')
         abbreviated_name = self.payload.get('abbreviated_name')
-        topic_domain.Topic.require_valid_name(name)
+        try:
+            topic_domain.Topic.require_valid_name(name)
+        except:
+            raise self.InvalidInputException(
+                'Invalid topic name.')
```

Comment:
Add info about what we received i.e, `Invalid topic name: %s`

Reply:
Done